api/src/pcapi/alembic/CONTRIBUTING.md

```diff
@@ -124,6 +123,243 @@ Note : le `select 1` permet d'éviter une requête vide, ce que SQLAlchemy n'app
 Attention : Ne pas ajouter de lignes vides à la fin du fichier alembic_version_conflict_detection.txt. L'ajout entraînera l'échec de l'étape de lint dans la CI sur la branche master une fois la pull request mergée.
 Si une ligne vide est ajoutée par erreur, il est nécessaire de merger une correction sur master, même si la CI échoue pour la pull request concernée.
 
+### Migrations inoffensives
+
+Ces migrations ne créeront (quasiment) pas de temps mort pour PostgreSQL 15.
+
+#### Ajout d'une colonne nullable
+
+À écrire dans une migration *pre* :
+
+```python
+op.add_column("venue", sa.Column("street", sa.Text(), nullable=True))
+```
+
+#### Ajout d'une colonne non-nullable avec une valeur constante par défaut
+
+À écrire dans une migration *pre* :
+
+```python
+op.add_column("product", sa.Column("isGcuCompatible", sa.BOOLEAN(), server_default=sa.text("true"), nullable=False))
+```
+
+Note : La valeur constante, ou non-volatile d'après le jargon PostgreSQL, est essentielle pour que cette migration soit inoffensive. `true`, `1` ou `''` sont des valeurs constantes, mais `now()` ne l'est pas.
+
+#### Ajout d'une valeur par défaut dans une colonne
+
+À écrire dans une migration *pre* :
+
+```python
+op.alter_column(
+    'offer_report',
+    'reportedAt',
+    existing_type=postgresql.TIMESTAMP(),
+    server_default=sa.text('now()'),
+)
+```
+
+La valeur par défaut ne va pas remplir la colonne des lignes existantes, que celles des futures lignes.
+
+#### Suppression d'une colonne
+
+À écrire dans une migration *post* :
```

Comment:
Il faut préciser que la colonne doit être nullable avant sa suppression (*). Si ça n'est pas le cas, il faut retirer la contrainte `not null` dans une migration **pré**-déploiement préalable (qui peut faire partie de la même MEP).

(*) Sinon le nouveau code pourrait faire des `insert` sans la colonne avant que la migration **post**-déploiement ait été exécutée, ce qui causera des "integrity error".